### PR TITLE
 Changing from Coalesce to OR Statement and removing extra space

### PR DESCRIPTION
**Pull request by Marymary-dev**

### CODE REVIEW

File: Scripts/REPORTING/load_aggregate_ushauri_appointments.sql [hunk 1]
```diff
@@ -133,7 +140,7 @@ Begin
                     Agencykey,
                     Agegroupkey
              From   ndwh.dbo.FactUshauriAppointments Sms
-             Where Tracinghomevisits=1
+             Where Tracinghomevisits> 0
```
Comment (nobert-mumo) [inline Scripts/REPORTING/load_aggregate_ushauri_appointments.sql:143]: @Marymary-dev for the metric `NumberOfHomeVisits` did we want to show total visits or total distinct patients who got home visits ?

File: Scripts/REPORTING/load_aggregate_ushauri_appointments.sql [hunk 2]
```diff
@@ -33,22 +33,29 @@ Begin
...
-             Where  Coalesce(Fourweeksmssent, Threeweeksmssent, Twoweeksmssent,
-                    Oneweeksmssent,
-                            Onedaysmssent) = 'Success'
-                    and Consentforsms = 'YES'
-             Group  By eomonth(cast(Appointmentdatekey as date)),
-                       Facilitykey,
-                       Partnerkey,
-                       Agencykey,
-                       Agegroupkey
+           SELECT
```
Comment (nobert-mumo) [inline Scripts/REPORTING/load_aggregate_ushauri_appointments.sql:36]: nice!

File: Scripts/REPORTING/load_aggregate_ushauri_appointments.sql (hunk 1 above)
Comment (Marymary-dev) [reply Scripts/REPORTING/load_aggregate_ushauri_appointments.sql:143]: > @Marymary-dev for the metric `NumberOfHomeVisits` did we want to show total visits or total distinct patients who got home visits ?

@nobert-mumo  It is the number of patients who were home visited

Comment (nobert-mumo) [reply Scripts/REPORTING/load_aggregate_ushauri_appointments.sql:143]: ah okay, maybe we can rename the column to smthing like `NoOfPatientswithHomeVisits`

**Comment by Marymary-dev** [comment]
Alright, this is done

File: Scripts/REPORTING/load_aggregate_ushauri_appointments.sql (hunk 1 above)
Comment (nobert-mumo) [reply Scripts/REPORTING/load_aggregate_ushauri_appointments.sql:143]: @Marymary-dev we would need to update the main select also here:
https://github.com/palladiumkenya/dwh-etl/pull/397/files#diff-8a9cfeb2c2e0f181655a2bdad6de6e92de573ee8e6e791680a0871a356681a27L289

**Comment by Marymary-dev** [comment]
ooh my, thanks, this is done

**Review by nobert-mumo — APPROVED** [review]
LGTM @Marymary-dev

**Pull request merged**